### PR TITLE
Add yaml-language-server

### PR DESCRIPTION
**Pull request by h-michael**

### CODE REVIEW

File: lua/nvim_lsp/yamlls.lua [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        new_config.cmd = {install_info.binaries[bin_name]}
+      end
+    end
+  end;
+  docs = {
+    vscode = "redhat.vscode-yaml";
+    description = [[
+https://github.com/redhat-developer/yaml-language-server
+
+`yaml-language-server` can be installed via `:LspInstall tsserver` or by yourself with `npm`:
```
Comment (mjlbach) [inline lua/nvim_lsp/yamlls.lua]: Should this be :LspInstall yamlls?

Comment (h-michael) [reply lua/nvim_lsp/yamlls.lua]: oops thanks :)

**Comment by justinmk** [comment]
needs a rebase to resolve conflict. (sorting the imports in `lua/nvim_lsp.lua` will reduce frequency of conflicts in the future...)

**Comment by h-michael** [comment]
@justinmk rebased

**Pull request merged**